tests/dragonfly/requirements.txt

```diff
@@ -14,3 +14,5 @@ pytest-asyncio==0.20.1
 pytest-repeat==0.9.1
 pymemcache==4.0.0
 numpy==1.24.3
+pytest-json-report
```

Comment:
Don't you have to specify at least some lower bound version requirement?

Reply:
I dont have to. I we dont specify a version the latest one will be taken.
Adding a version is for compatibility issues for later version.
I will add a version